clusters/build-clusters/02_cluster/openshift-machine-api/pthread-debug_mcp.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfigPool
 metadata:
-  name: pthread-debug
+  name: pd
   labels:
-    pthread-debug: "true"
+    pd: "true"
 spec:
   machineConfigSelector:
     matchExpressions:
-      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,pthread-debug]}
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,pd]}
```

Comment:
what? We only want that for the pthread instance so the rest remains without this fix so we can test that the issue does occur on the non-fix instances and does not occur on the fix instances
/hold

Reply:
I understood this.
I need to make the `containerruntimeconfig` take effects (`pidsLimit: 0`) first, right?